### PR TITLE
MONEY-474

### PR DESCRIPTION
**Pull request by mtowers**
* use `activity.published` attribute for TTL indexes

### CODE REVIEW

File: lib/realself/feed/ttl.rb [hunk 1]
```diff
@@ -15,61 +14,15 @@ def ensure_index(owner_type, background: true)
...
-
-
-      ##
-      # Insert a stream activity in to a ttl feed
-      #
-      # @param [ Objekt ] the owner of the feed
-      # @param [ StreamActivity ] the stream activity to insert in to the feed
-      def insert(owner, stream_activity)
-        activity_hash = stream_activity.to_h
-        activity_hash[:created] = Time.now.utc
```
Comment (yqset) [inline lib/realself/feed/ttl.rb]: 👍

File: lib/realself/feed/ttl.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 module RealSelf
   module Feed
-    class Ttl
-      include Getable
+    class Ttl < Permanent
```
Comment (yqset) [inline lib/realself/feed/ttl.rb]: Nice!

**Comment by yqset** [comment]
:shipit:

File: spec/integration/realself/feed/ttl_spec.rb [hunk 3]
```diff
@@ -49,7 +49,7 @@ class TestTtl < RealSelf::Feed::Ttl
             'object.id'      => Mongo::Index::DESCENDING})
           expect(index[:unique]).to eql true
           index[:name]
-        when "created_1"
+        when "activity.published_-1"
```
Comment (wulftone) [inline spec/integration/realself/feed/ttl_spec.rb]: looks funny, i assume it's correct?

**Comment by yqset** [comment]
Looks good!

**Pull request merged**